Source/Core/Core/ConfigManager.cpp

```diff
@@ -195,23 +195,10 @@ void SConfig::SaveCoreSettings(IniFile& ini)
   core->Set("CustomRTCValue", m_customRTCValue);
 }
 
-void SConfig::SaveUSBPassthroughSettings(IniFile& ini)
-{
-  IniFile::Section* section = ini.GetOrCreateSection("USBPassthrough");
-
-  std::ostringstream oss;
-  for (const auto& device : m_usb_passthrough_devices)
-    oss << fmt::format("{:04x}:{:04x}", device.first, device.second) << ',';
-  std::string devices_string = oss.str();
-  if (!devices_string.empty())
-    devices_string.pop_back();
-
-  section->Set("Devices", devices_string);
-}
-
 void SConfig::LoadSettings()
 {
   Config::Load();
+  IOS::HLE::LoadUSBWhitelistFromConfig();
```

Comment:
I'm... not sure where to put this, honestly. This doesn't seem like a good place for it, but I don't know of a better one.